src/openrct2/localisation/StringIds.h

```diff
@@ -3982,7 +3982,9 @@ enum : uint16_t
     STR_TRACK_DESIGN_BLOCK_BRAKE_SPEED_RESET = 6579,
 
     STR_RESET = 6580,
-    STR_RESET_SHORTCUT_KEYS_PROMPT = 6581
+    STR_RESET_SHORTCUT_KEYS_PROMPT = 6581,
+
+    STR_SHORTCUT_OPEN_KEYBOARD_SHORTCUTS_WINDOW = 6582
```

Comment:
Maybe we should end he last element also with a comma otherwise we have to always modify the line prior.

Reply:
I added the trailing comma.